<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>653</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextEdit" name="te_input_asm"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>addr:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edit_asm_addr">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>41</height>
             </size>
            </property>
            <property name="text">
             <string>40000</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_keystone_asm">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>41</height>
             </size>
            </property>
            <property name="text">
             <string>keystone asm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="tb_output_asm"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="te_input_dsm"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>addr:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edit_dsm_addr">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>41</height>
             </size>
            </property>
            <property name="text">
             <string>40000</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_capstone_dsm">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>41</height>
             </size>
            </property>
            <property name="text">
             <string>capstone asm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="tb_output_dsm"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1023</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
